rocketpy/tools.py

```diff
@@ -1231,6 +1231,42 @@ def find_closest(ordered_sequence, value):
     return pivot_index - 1 if value - smaller <= greater - value else pivot_index
 
 
+def import_optional_dependency(
+    name,
+):
+    """Import an optional dependency.
+
+    github.com/pandas-dev/pandas/blob/main/pandas/compat/_optional.py
+    If the dependency is not installed, an ImportError is raised.
```

Comment:
There seems to be an issue with this docstring, perhaps part of the first phrase that leads to the pandas link.

I would write something along the lines "based on the implementation found in ...".

Reply:
Solved: [7cf2435](https://github.com/RocketPy-Team/RocketPy/pull/403/commits/7cf2435ed069f331608bafd471081888d3e40ad4)